events/test.py

```diff
@@ -0,0 +1,92 @@
+import unittest
+import datetime
+import app
+
+
+class TestEventsDB(unittest.TestCase):
```

Comment:
You might want to use the `mongomock` (which is actually a fake) that Gabe found in #2.

Reply:
Ref for differences between Mocks, Fakes, and Stubs:

https://martinfowler.com/articles/mocksArentStubs.html#TheDifferenceBetweenMocksAndStubs